### ISSUE TITLE
Wrong bulk CanoGamez2020 files

### ISSUE THREAD

**Issue by AnthoOzier** [neurorestore/DE-analysis#7]
Hello again, 

I am trying to reproduce your analyses on the 18 'ground truth' datasets provided with your paper on Zenodo (https://doi.org/10.5281/zenodo.5048449) and I noticed that the bulk files associated to the CanoGamez2020 publication all contain the same assay slot but different meta slots. 

Each file contains a copy of the assay of the "CanoGamez2020_memory-iTreg" dataset, which is the only file where sample_ids in the meta slot match with the sample_ids of the assay table. 
  
In other words, the metadata are correct but not the data for the bulk CanoGamez2020 files (except CanoGamez2020_memory-iTreg dataset). 
Would it be possible for you to provide the correct datasets for the CanoGamez files ?

Thank you in advance.

**Comment by skinnider** [comment]
Thanks for catching this. Tagging @jordansquair to update the Cano-Gamez datasets on Zenodo.

**Comment by jordansquair** [comment]
CanoGamez datasets are updated on Zenodo.
